src/common/hf.ts

```diff
@@ -111,8 +111,8 @@ export const values = async (opts?: ValuesOptions): Promise<Record<string, any>>
   return opts?.replacePath ? value.rp : value.clean
 }
 
-export const hfValues = async (skipCache = false): Promise<Record<string, any>> => {
-  return values({ replacePath: true, skipCache })
+export const hfValues = async (opts?: Omit<ValuesOptions, 'replacePath'>): Promise<Record<string, any>> => {
```

Comment:
As discussed with @Dunky13 we should switch these names:

- `hfValues` should be named `values`, and may contain logic/abstractions to shield the user from knowing where values come from (chart or `helmfile values`)
- `values` should be named `hfValues` as that is the function using helmfile.

Reply:
We discussed it, but also came to the conclusion to let it be for now, as it would require more than just refactoring naming. But as you suggest, also logic.

Reply:
That is not true, as it could just be renaming...

Reply:
not part of this PR

Reply:
what PR is it part of then? Don't make such statements please, as it is not productive...just rename them as it is part of this work